spec/models/refinery/jobs/job_spec.rb

```diff
@@ -1,10 +1,16 @@
 require 'spec_helper'
+require 'support/helper/generator_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.